saleor/core/management/commands/update_search_indexes.py

```diff
@@ -15,7 +15,7 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         # Update products
-        products_total_count = Product.objects.filter(search_document="").count()
+        products_total_count = Product.objects.filter(search_vector=None).count()
```

Comment:
This can cause memory issues on big databases. Please consider to reduce BATCH size in saleor/core/search_tasks.py

Reply:
There is no counting any more.